Make.config

```diff
@@ -453,6 +453,15 @@ else
 DOTNET5=$(abspath $(TOP)/builds/downloads/dotnet/$(DOTNET5_VERSION))/dotnet
 endif
 
+DOTNET_IOS_RUNTIME_IDENTIFIERS=ios-arm ios-arm64 ios-x86 ios-x64
+DOTNET_TVOS_RUNTIME_IDENTIFIERS=tvos-arm64 tvos-x64
+DOTNET_WATCHOS_RUNTIME_IDENTIFIERS=watchos-arm watchos-x86
```

Comment:
what about `arm64_32` ?

Reply:
The runtime identifiers for watchOS are still unclear, but my guess is that `watchos-arm` will contain both `armv7k` and `arm64_32` (like `ios-arm` contains both `armv7` and `armv7s`).